src/core/auth.ts

```diff
@@ -63,16 +72,44 @@ export const auth = (client: ApolloClient<NormalizedCacheObject>): AuthSDK => {
   /**
    * Registers user with email and password.
    *
-   * @param opts - Object with user's data. Email, password, redirectUrl and channel are required fields.
+   * @param opts - Object with user's data. Email and password are required fields.
+   * "channel" can be changed by using first "changeChannel" method from api.
    * @returns Promise resolved with AccountRegister type data
    */
   const register: AuthSDK["register"] = async opts =>
-    await client.mutate({
+    await client.mutate<RegisterMutation, RegisterMutationVariables>({
       mutation: REGISTER,
       variables: {
         ...opts,
+        channel,
       },
     });
 
-  return { login, logout, register };
+  /**
+   * Refresh JWT token. If it fails it will try to take refreshToken
+   * from the http-only cookie refreshToken.
+   *
+   * @param opts - Optional object with csrfToken and refreshToken. It is required when refreshToken is provided as a cookie.
```

Comment:
What is required? csrfToken? This message could be a bit more clear.

Reply:
From our docs https://docs.saleor.io/docs/3.0/developer/api-reference/mutations/token-refresh/#refreshtoken-string
```
Refresh JWT token. Mutation tries to take refreshToken from the input.If it fails it will try to take refreshToken from the http-only cookie -refreshToken. csrfToken is required when refreshToken is provided as a cookie.
```